### PR TITLE
February update - part 1

### PR DESCRIPTION
**Pull request by pet-mit**

### CODE REVIEW

File: pages/docs/parameters/rao-parameters.md [hunk 1]
```diff
@@ -0,0 +1,754 @@
...
+- **Expected value**: true/false
+- **Default value**: false
+- **Usage**: if this parameter is set to true, the RAO will detect TSOs not sharing any curative remedial actions (in
+  the CRAC). During the curative RAO, these TSOs' CNECs will not be taken into account in the minimum margin objective
+  function, unless the applied curative remedial actions decrease their margins (compared to their margins before 
+  applying any curative action).  
+  If it is set to false, all CNECs are treated equally in the curative RAO.  
+  This parameter has no effect on the preventive RAO.  
+  This parameter should be set to true for CORE CC.
+  This parameter is not compatible with [do-not-optimize-cnec-secured-by-its-pst](#do-not-optimize-cnec-secured-by-its-pst).
```
Comment (phiedw) [inline pages/docs/parameters/rao-parameters.md:372]: for technical reasons

File: pages/docs/parameters/rao-parameters.md [hunk 2]
```diff
@@ -0,0 +1,754 @@
...
+- **Expected value**: a map with string keys and values. The keys should represent critical network element IDs, and the
+  values should represent PST network element IDs.
+- **Default value**: empty map
+- **Usage**: when a critical network element (identified by its PowSyBl ID) is associated to a PST (identified by its
+  PowSyBl ID) in this parameter, CNECs defined on the critical element will not be taken into account in the minimum
+  margin objective function, as long as they can be secured by the associated PST. In other words, they will only be
+  taken into account if the PST has too few tap positions left to reduce the flow constraints on these CNECs.  
+  This parameter affects both preventive and curative RAOs.  
+  It is actually used for the SWE CC process.
+  This parameter is not compatible with [do-not-optimize-curative-cnecs-for-tsos-without-cras](#do-not-optimize-curative-cnecs-for-tsos-without-cras).
```
Comment (phiedw) [inline pages/docs/parameters/rao-parameters.md:384]: for technical reasons

File: pages/docs/input-data/crac/creation-context.md [hunk 3]
```diff
@@ -0,0 +1,542 @@
...
+optimized nor monitored, etc.)
+- **[ADDED]**: happens if FARAO decides to add elements that were not explicitly defined in the original file (e.g. if the 
+CRAC contains AUTO CNECs without any remedial action associated, FARAO will automatically duplicate them in the OUTAGE 
+instant in order to secure them during the preventive RAO)
+- **[ALTERED]**: happens if FARAO imports an element after altering it or ignoring some of its components (e.g. if a monitored 
+element shall be so after multiple contingencies, among which some were not imported for any reason, then only valid 
+contingencies will be used for the created CNECs)
+- **[WARN]**: non-critical warnings (e.g. if the user defined a timestamp for a CRAC format tht doesn't require one, the 
+timestamp is ignored and a warning is logged)
+- **[INFO]**: non-critical information
```
[the remark this answers is not in the document]
Comment (pet-mit) [reply pages/docs/input-data/crac/creation-context.md:81]: we actually don't have any

File: pages/docs/input-data/crac/creation-parameters.md [hunk 4]
```diff
@@ -0,0 +1,338 @@
...
+The CSE native CRAC format does not allow defining [aligned range actions](crac#range-action). This extra parameter 
+allows the user to do just that.  
+To use it, you have to define a list of strings containing the IDs of range actions that have to be aligned seperated by a 
+" + " sign; for example "range-action-1-id + range-action-17-id" and "range-action-8-id + range-action-9-id".  
+See [example below](#cse-example) for a better illustration.
+
+### bus-bar-change-switches {#bus-bar-change-switches}
+As explained in the CSE native CRAC format section [here](cse#bus-bar), bus-bar-change remedial actions are defined in FARAO 
+as [switch pair network actions](crac#switch-pair).  
+These switches are not defined in the native CRAC not in the original network, they should be created artificially in the
```
[the remark this answers is not in the document]
Comment (pet-mit) [reply pages/docs/input-data/crac/creation-parameters.md:101]: These switches are not defined in the native CRAC nor* in the original network

File: pages/docs/input-data/crac/json.md [hunk 5]
```diff
@@ -48,192 +51,242 @@ example on github
...
 
-[Utility methods](creation-methods) have been developed in FARAO to ease the management of the Imax and nominal voltage during the creation of FlowCnecs.
-
-
-A FlowCnec has a reliability margin, also known as FRM for Flow Reliability Margin. The reliability margin can only be defined in megawatt. It is subtracted from the min/max values of the FlowCnec thresholds when the limits of the FlowCnec are computed.
-
-##### Create a CNEC
+> 💡  **NOTE**
+> A FlowCnec's reliability margin is also known as FRM for Flow Reliability Margin. 
+> It can only be defined in megawatt, it is subtracted from the thresholds of the FlowCnec, adding extra an constraint.
```
[the remark this answers is not in the document]
Comment (pet-mit) [reply pages/docs/input-data/crac/json.md:287]: adding an extra constraint*
 (I meant that it reduces thresholds, should I add this explanation?)

File: pages/docs/input-data/crac/json.md [hunk 6]
```diff
@@ -368,7 +427,11 @@ In FARAO, AngleCnecs can be created by the java API, or written in the json CRAC
   .withOperator("cnec1Operator")
   .withExportingNetworkElement("eneId1", "eneName1")
   .withImportingNetworkElement("ineId1", "ineName1")
-  .newThreshold().withUnit(Unit.DEGREE).withMax(1000.0).withMin(-1000.0).add()
+  .newThreshold()
+    .withUnit(Unit.DEGREE)
+    .withMax(1000.0)
+        .withMin(-1000.0)
```
[the remark this answers is not in the document]
Comment (pet-mit) [reply pages/docs/input-data/crac/json.md:433]: no

File: pages/docs/input-data/crac/json.md [hunk 7]
```diff
@@ -768,14 +887,22 @@ TapRanges can be of different types:
...
 {% capture t9_java %}
 ~~~java
 crac.newPstRangeAction()
-    .withId("pst-range-action-id")
-    .withName("pst-range-action-name")
+    .withId("pst-range-action-1-id")
+    .withName("pst-range-action-1-name")
     .withOperator("operator")
     .withNetworkElement("pst-network-element-id")
+    .withGroupId("pst-range-action-1 is aligned with pst-range-action-2")
```
[the remark this answers is not in the document]
Comment (pet-mit) [reply pages/docs/input-data/crac/json.md:905]: I thought the same, that's why the "pst-range-action-1" I used is neither the ID not the name

[the remark this answers is not in the document]
Comment (pet-mit) [reply pages/docs/input-data/crac/json.md:905]: How about I add this to the groupId field description (in the 3rd tab): 
You can use any group ID you like, as long as you use the same for all the range actions you want to align

File: pages/docs/parameters/rao-parameters.md [hunk 8]
```diff
@@ -0,0 +1,754 @@
...
+### acceptable-margin-decrease {#mnec-acceptable-margin-decrease}
+- **Expected value**: numeric values, in MEGAWATT unit
+- **Default value**: 50 MW (required by CORE CC methodology)
+- **Usage**: the decrease of the initial margin that is allowed by the optimisation on MNECs.  
+  In other words, it defines the bounds for the margins on the MNECs by  
+  *Mcnec ≥ max(0, m0cnec − acceptableDiminution)*  
+  With *Mcnec* the margin on the cnec after optimisation, *m0cnec* the initial margin on the cnec, and 
+  *acceptableDiminution* the so-called "acceptable-margin-decrease" coefficient.  
+  For the CORE CC calculation, the ACER methodology fixes this coefficient at 50 MW.  
+  For CSE CC calculation, setting this parameter to -99999 allows the MNEC constraints to consider
```
[the remark this answers is not in the document]
Comment (pet-mit) [reply pages/docs/parameters/rao-parameters.md:525]: as opposed to considering CRAC threshold + initial MNEC flow

File: pages/docs/input-data/crac/creation-context.md [hunk 9]
```diff
@@ -0,0 +1,542 @@
...
+tags: [Docs, Data, CRAC]
+---
+
+## Introduction {#introduction}
+When FARAO tries to import a native CRAC file ([FlowBasedConstraint](fbconstraint.md), [CSE](cse.md), [CIM](cim.md), ...) 
+into an [internal CRAC format](json.md), some data transformation can happen, and data present in the final CRAC object 
+will not be a "one-to-one" exact representation of the data in the original file.  
+This can be an issue for the final user, as [querying the RAO result file or object](/docs/output-data/rao-result-json#contents) 
+needs knowledge of the artefacts FARAO created during CRAC creation.  
+The [CracCreationContext](https://github.com/farao-community/farao-core/blob/master/data/crac-creation/crac-creator-api/src/main/java/com/farao_community/farao/data/crac_creation/creator/api/CracCreationContext.java)
```
[the remark this answers is not in the document]
Comment (pet-mit) [reply pages/docs/input-data/crac/creation-context.md:19]: adding a custom image would be time consuming. I can easily add an emoji to all external links, for example this one: ↗️
or, better, a unicode character like this one: ↗

**Comment by pet-mit** [comment]
included in [next PR](https://github.com/farao-community/farao-community.github.io/pull/56)

**Pull request closed without merging**